GetEligibilityCheck/handler.ts

```diff
@@ -0,0 +1,96 @@
+import { Context } from "@azure/functions";
+import * as df from "durable-functions";
+import * as express from "express";
+import { fromNullable } from "fp-ts/lib/Option";
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { FiscalCodeMiddleware } from "io-functions-commons/dist/src/utils/middlewares/fiscalcode";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "io-functions-commons/dist/src/utils/request_middleware";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import {
+  IResponseErrorInternal,
+  IResponseSuccessAccepted,
+  IResponseSuccessJson,
+  ResponseErrorInternal,
+  ResponseSuccessAccepted,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+import { ActivityResultSuccess } from "../EligibilityCheckActivity/handler";
+import { SottoSogliaEnum } from "../generated/definitions/ConsultazioneSogliaIndicatoreResponse";
+import { EligibilityCheck } from "../generated/definitions/EligibilityCheck";
+import { EligibilityCheckStatusEnum } from "../generated/definitions/EligibilityCheckStatus";
+import { NucleoType } from "../generated/definitions/NucleoType";
+import { initTelemetryClient } from "../utils/appinsights";
+
+type IGetEligibilityCheckHandler = (
+  context: Context,
+  fiscalCode: FiscalCode
+) => Promise<
+  | IResponseSuccessJson<EligibilityCheck>
+  | IResponseErrorInternal
+  | IResponseSuccessAccepted
+>;
+
+initTelemetryClient();
+
+function calculateBonus(familyMembers: ReadonlyArray<NucleoType>): number {
+  return familyMembers.length > 2
+    ? 50000
+    : familyMembers.length === 2
+    ? 25000
+    : familyMembers.length === 1
+    ? 15000
+    : 0;
+}
+
+// tslint:disable-next-line: cognitive-complexity
+export function GetEligibilityCheckHandler(): IGetEligibilityCheckHandler {
+  return async (context, fiscalCode) => {
+    const client = df.getClient(context);
+    const status = await client.getStatus(fiscalCode);
+    if (status.runtimeStatus === df.OrchestrationRuntimeStatus.Running) {
+      return ResponseSuccessAccepted("Orchestrator already running");
+    }
+    return ActivityResultSuccess.decode(status.customStatus)
+      .map(_ => {
+        const bonusValue = fromNullable(_.data.Componente)
+          .map(calculateBonus)
+          .getOrElse(0);
+        return EligibilityCheck.encode({
+          family_members: _.data.Componente || [],
+          max_amount: bonusValue,
```

Comment:
@gunzip 
@BurnedMarshal
Why we return max_amount and max_tax_benefit also if the user ISEE is INELIGIBLE?

Reply:
we can remove the properties or return `0` as value for `INELIGIBLE` ISEE

Reply:
yes, we must review the required param. we'll do it with another pr. by now assume every param is required